user-docs/documentation/docs/how-to-guides/5-deploy-cdk-constructs.md

```diff
@@ -30,33 +32,38 @@ class MyConstruct extends Construct {
 }
 
 const serverlessConfiguration: AWS & ServerlessCdkPluginConfig = {
-  service: "my-cdk-construct-service",
+  service: "my-cdk-service",
 
   plugins: [
-    ...
+    // ...
```

Comment:
Why add the comments?

Reply:
The idea was to allow the snippet to be copy pasted, and have comments act as placeholders rather than breaking the syntax.

Reply:
oki, fine by me